### DISCUSSION TITLE
Bump `botocore` & remove restriction

### DISCUSSION THREAD

**Discussion by kaxil** [aio-libs/aiobotocore#920] [Q&A] [answered]
Hi team,

[Apache Airflow](https://github.com/apache/airflow) Committer here 👋 , I would like to propose that we remove the pinned version of `botocore` if there is no specific reason (please correct me if there is a reason).

https://github.com/aio-libs/aiobotocore/blob/40215851cbc48b01428ae43565d827fa8436d040/setup.py#L10

In the Airflow project, we provide a set of known working Constraints files (for each python version) with every Airflow release described in https://github.com/apache/airflow#installing-from-pypi

In general, if you restrict the version of botocore, it makes it very difficult to satisfy dependencies from all the other projects.

Context: We are building new Airflow Operators leveraging Python's Async framework and using `aiobotocore` lib (thanks for all the work here) to interact with AWS services. 

I understand if you pin the major version for `botocore` but is there a reason we can't bump this dependency?

**Comment by kaxil** [comment, marked as answer]
`2.2.0` was released y'day that fixes this issue - https://github.com/aio-libs/aiobotocore/blob/master/CHANGES.rst#220-2022-03-16

**Comment by thehesiod** [comment]
aiobotocore is highly coupled to botocore as we override several methods so we have to release it in sync.
